### DISCUSSION TITLE
开源之道（2016.6.3） - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#106] [Announcements]
# 开源之道（2016.6.3） - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2016/2016_6/2016_6_3/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

为什么开源...这几天 Musk 刚刚决定因为开除太多工程师而无法维护的 Twitter 代码,
现在看, 真心扎心...

忽然有了这么多分享的消息, 应该是找到了 RSS 源?
